src/ucp/api/ucp.h

```diff
@@ -2824,6 +2855,42 @@ ucs_status_ptr_t ucp_stream_recv_nb(ucp_ep_h ep, void *buffer, size_t count,
                                     size_t *length, unsigned flags);
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Non-blocking stream receive operation of structured data into a
+ *        user-supplied buffer.
+ *
+ * This routine receives data that is described by the local address @a buffer,
+ * size @a count object on the endpoint @a ep. The routine is non-blocking
+ * and therefore returns immediately. The receive operation is considered
+ * complete when the message is delivered to the buffer. If the receive
+ * operation cannot be started, the routine returns an error.
+ *
+ * @param [in]     ep       UCP endpoint that is used for the receive operation.
+ * @param [in]     buffer   Pointer to the buffer to receive the data to.
+ * @param [in]     count    Number of elements to receive into @a buffer.
+ * @param [out]    length   Size of the received data in bytes. The value is
+ *                          valid only if return code is UCS_OK.
+ * @note                    The amount of data received, in bytes, is always an
+ *                          integral multiple of the @a datatype size.
```

Comment:
move this not after params/return value doc text

Reply:
moved upper... but I just copy-pasted this text from ucp_stream_recv_nb description - there is same wording/format

Reply:
i think it should be moved lower (after `@return otherwise`)

Reply:
ok, moved down